.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,32 @@
+---

```

Comment:
I think this is not needed in this commit

Reply:
Well, at least separate PR. To be honest I dont think these files are needed at this point, maybe much later.